components/.eslintrc.js

```diff
@@ -7,8 +7,7 @@ module.exports = {
     'plugin:vue/vue3-essential',
     'eslint:recommended',
     '@vue/typescript/recommended',
-    '@vue/prettier',
-    '@vue/prettier/@typescript-eslint',
+    'plugin:prettier/recommended',
```

Comment:
`@vue/eslint-config-prettier` package remove due to deprecation. `eslint-plugin-vue` is added.

https://cli.vuejs.org/migrations/migrate-from-v4.html#eslint-plugin